components/nexusoperations/workflow/commands_test.go

```diff
@@ -533,7 +537,7 @@ func TestHandleCancelCommand(t *testing.T) {
 
 	t.Run("operation already completed - completion buffered", func(t *testing.T) {
 		tcx := newTestContext(t, defaultConfig)
-		tcx.ms.EXPECT().HasAnyBufferedEvent(gomock.Any()).Return(true)
+		tcx.ms.EXPECT().HasAnyBufferedEvent(gomock.Any()).Return(true).AnyTimes() // Changed to AnyTimes()
```

Comment:
Hmm... why?

And please remove the trailing comment.

Reply:
Ah, we changed the prod code in our pair programming. Fixed the test. Thanks!